gui/fitpanel/src/TFitEditor.cxx

```diff
@@ -2181,11 +2182,10 @@ void TFitEditor::DoFit()
 
    // Save fit data for future use as a PrevFit function.
    TF1* tmpTF1 = copyTF1(fitFunc);
-   ostringstream name;
-   name << "PrevFit-" << fPrevFit.size() + 1;
-   if ( strcmp(tmpTF1->GetName(), "PrevFitTMP") != 0 )
-      name << "-" << tmpTF1->GetName();
-   tmpTF1->SetName(name.str().c_str());
+   TString name = TString::Format("PrevFit-%d", (int) fPrevFit.size() + 1);
+   if (!strstr(fitFunc->GetName(),"PrevFit"))
```

Comment:
Why did we need to check for `"PrevFitTMP"` before? Was this simply a bug? What does `git blame` say - did we use `"PrevFitTMP"` some time in the past?

Reply:
"PrevFitTMP" is name of object created right before fit.
But than it was renamed into something like "PrevFit-3". 
If next time such function was used for fitting, result name was "PrevFit-5-PrevFit-3".
And so on.